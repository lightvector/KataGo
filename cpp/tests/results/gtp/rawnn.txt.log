: Running with following config:
allowResignation = true
cudaUseFP16 = false
cudaUseNHWC = false
forDeterministicTesting = true
lagBuffer = 1.0
logAllGTPCommunication = true
logFile = tests/results/gtp/rawnn.txt.log
logSearchInfo = true
logSearchInfoForChosenMove = false
logTimeStamp = false
logToStderr = false
maxPlayouts = 10000
maxTimePondering = 60.0
maxVisits = 100
nnRandSeed = forTesting
nnRandomize = false
numSearchThreads = 1
openclUseFP16 = false
ponderingEnabled = false
resignConsecTurns = 3
resignThreshold = -0.90
rootSymmetryPruning = false
rules = tromp-taylor
searchFactorAfterOnePass = 0.50
searchFactorAfterTwoPass = 0.25
searchFactorWhenWinning = 0.40
searchFactorWhenWinningThreshold = 0.95
searchRandSeed = forTesting
trtUseFP16 = false

: GTP Engine starting...
: KataGo v1.16.4
: Using TrompTaylor rules initially, unless GTP/GUI overrides this
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 19 * 19 exactly
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Initializing board with boardXSize 19 boardYSize 19
: Loaded config configs/gtp_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: logDir = 
: Config override: logFile = tests/results/gtp/rawnn.txt.log
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: nnRandSeed = forTesting
: Config override: nnRandomize = false
: Config override: numSearchThreads = 1
: Config override: openclUseFP16 = false
: Config override: rootSymmetryPruning = false
: Config override: searchRandSeed = forTesting
: Config override: trtUseFP16 = false
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: boardsize 9
: GPU -1 finishing, processed 0 rows 0 batches
: Cleaned up old neural net and bot
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 9 * 9 exactly
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 9 nnYLen 9
: Initializing board with boardXSize 9 boardYSize 9
: = 
: Controller: kata-set-rules chinese
: Changed rules to Chinese
: = 
: Controller: play b c3
: = 
: Controller: play w c6
: = 
: Controller: kata-raw-nn 1
: = symmetry 1
whiteWin 0.532987
whiteLoss 0.466952
noResult 0.000062
whiteLead 0.577
whiteScoreSelfplay 0.427
whiteScoreSelfplaySq 224.101
varTimeLeft -1.000
shorttermWinlossError -1.000
shorttermScoreError -1.000
policy
0.000016 0.000013 0.000014 0.000015 0.000018 0.000016 0.000014 0.000016 0.000013 
0.000017 0.000028 0.000093 0.000148 0.000223 0.000055 0.000026 0.000024 0.000017 
0.000020 0.000066 0.000677 0.008790 0.099234 0.063623 0.000615 0.000026 0.000015 
0.000015 0.000238     NAN 0.106602 0.188578 0.295184 0.006745 0.000030 0.000019 
0.000016 0.000191 0.000806 0.008210 0.007017 0.096975 0.006814 0.000041 0.000022 
0.000017 0.000041 0.000046 0.000334 0.007134 0.091387 0.004063 0.000034 0.000019 
0.000023 0.000022     NAN 0.000039 0.000640 0.004052 0.000397 0.000029 0.000016 
0.000015 0.000020 0.000023 0.000028 0.000036 0.000039 0.000031 0.000025 0.000021 
0.000009 0.000012 0.000017 0.000015 0.000016 0.000015 0.000013 0.000017 0.000014 
policyPass 0.000001 
whiteOwnership
0.2936487 0.2919934 0.1909822 0.0572251 -0.0237669 -0.0365063 -0.0872653 -0.0864983 -0.0678094 
0.3363636 0.3862071 0.1751917 0.1181649 -0.0842159 -0.0390507 -0.0837802 -0.0592106 -0.0673248 
0.3707412 0.4659901 0.4427463 0.0712150 -0.2949021 -0.2894870 -0.1729886 -0.1012077 -0.0657026 
0.3219600 0.4218068 0.6774654 0.0049667 -0.2693184 -0.4080397 -0.2728807 -0.1312715 -0.0227591 
0.1393000 0.1681466 0.1606477 -0.0556220 -0.1733975 -0.4068947 -0.2896566 -0.1382011 -0.0732938 
-0.1477411 -0.0294894 -0.1123110 -0.1720132 -0.2981643 -0.3713466 -0.2319424 -0.1367578 -0.0733546 
-0.3615263 -0.4255128 -0.6879031 -0.2779490 -0.2943310 -0.2443602 -0.1282024 -0.1103746 -0.1024175 
-0.5447001 -0.5222596 -0.5039542 -0.3954769 -0.3323049 -0.2144439 -0.2660975 -0.1903932 -0.0947189 
-0.4543642 -0.5465777 -0.4031759 -0.3866568 -0.2783843 -0.2601057 -0.2542107 -0.2195328 -0.1205025
: Controller: kata-raw-nn 4
: = symmetry 4
whiteWin 0.525422
whiteLoss 0.474523
noResult 0.000055
whiteLead 0.408
whiteScoreSelfplay 0.366
whiteScoreSelfplaySq 209.486
varTimeLeft -1.000
shorttermWinlossError -1.000
shorttermScoreError -1.000
policy
0.000014 0.000017 0.000017 0.000012 0.000017 0.000017 0.000013 0.000016 0.000018 
0.000018 0.000036 0.000083 0.000100 0.000136 0.000044 0.000024 0.000025 0.000018 
0.000019 0.000041 0.000495 0.003076 0.096997 0.073348 0.000566 0.000026 0.000015 
0.000016 0.000149     NAN 0.205962 0.241965 0.283258 0.006111 0.000035 0.000017 
0.000015 0.000107 0.000522 0.004869 0.008021 0.047599 0.003643 0.000043 0.000020 
0.000017 0.000028 0.000047 0.000185 0.001472 0.017526 0.001585 0.000036 0.000019 
0.000020 0.000023     NAN 0.000034 0.000197 0.000752 0.000104 0.000025 0.000016 
0.000017 0.000016 0.000018 0.000024 0.000026 0.000028 0.000027 0.000023 0.000017 
0.000012 0.000019 0.000019 0.000018 0.000017 0.000021 0.000017 0.000018 0.000014 
policyPass 0.000001 
whiteOwnership
0.1924710 0.2665748 0.1818978 0.1237667 0.0040299 0.0171145 -0.0698032 -0.0827631 -0.0664251 
0.3655786 0.4113192 0.1882543 0.1307335 -0.0316230 -0.0697506 -0.1127632 -0.0177444 -0.0810781 
0.3159997 0.3355140 0.3357060 0.0135291 -0.2981014 -0.3415389 -0.1793087 -0.0971713 -0.0966877 
0.2204663 0.4383409 0.6254903 -0.0182784 -0.3503966 -0.4122772 -0.2190450 -0.1177377 -0.0559397 
-0.0493255 -0.0195753 0.0772057 -0.0930304 -0.1736475 -0.3594979 -0.2188754 -0.1123595 -0.0658549 
-0.2297983 -0.1495231 -0.1323323 -0.0070033 -0.2083750 -0.2728684 -0.1630310 -0.0722803 -0.0409004 
-0.3641602 -0.4937686 -0.7472466 -0.3225802 -0.2023943 -0.2291806 -0.1735636 -0.0958892 -0.0523615 
-0.4939024 -0.5195467 -0.5189104 -0.4030693 -0.3010308 -0.2836112 -0.1641537 -0.0888866 -0.1160190 
-0.4077278 -0.5694960 -0.4338544 -0.3839099 -0.3046613 -0.1598477 -0.1567573 -0.0958614 -0.0911347
: Controller: kata-raw-nn 6
: = symmetry 6
whiteWin 0.549882
whiteLoss 0.450066
noResult 0.000052
whiteLead 1.019
whiteScoreSelfplay 1.026
whiteScoreSelfplaySq 223.759
varTimeLeft -1.000
shorttermWinlossError -1.000
shorttermScoreError -1.000
policy
0.000016 0.000013 0.000015 0.000013 0.000019 0.000016 0.000014 0.000017 0.000011 
0.000015 0.000028 0.000092 0.000214 0.000272 0.000060 0.000023 0.000032 0.000016 
0.000014 0.000052 0.000733 0.007403 0.088516 0.057368 0.000598 0.000026 0.000015 
0.000012 0.000112     NAN 0.141371 0.216024 0.336074 0.006142 0.000032 0.000016 
0.000013 0.000087 0.000699 0.002043 0.001808 0.075195 0.007399 0.000041 0.000019 
0.000013 0.000033 0.000044 0.000192 0.002116 0.048740 0.002915 0.000034 0.000016 
0.000015 0.000022     NAN 0.000040 0.000337 0.002158 0.000253 0.000027 0.000013 
0.000013 0.000020 0.000015 0.000026 0.000032 0.000030 0.000025 0.000020 0.000016 
0.000011 0.000014 0.000015 0.000017 0.000017 0.000019 0.000016 0.000018 0.000013 
policyPass 0.000001 
whiteOwnership
0.2329700 0.2496214 0.1514244 0.1106936 0.0183179 0.0054265 -0.0272415 -0.0068095 -0.0624254 
0.3365522 0.3701028 0.1809798 0.1033716 -0.0579549 -0.0847751 -0.1283866 -0.0175767 -0.0841713 
0.3632700 0.4063769 0.3204426 0.0934118 -0.3370920 -0.2827722 -0.1681476 -0.0848208 -0.0891701 
0.2925220 0.4161210 0.6190414 0.0766588 -0.3506812 -0.4414802 -0.2614721 -0.0942562 -0.0398460 
0.0639914 0.0637949 0.0144126 -0.0352721 -0.2306587 -0.4197260 -0.3087422 -0.1215060 -0.0756596 
-0.1826633 -0.0298219 -0.0936258 -0.0887200 -0.2047295 -0.3158493 -0.1682725 -0.0682627 -0.0753705 
-0.3675333 -0.4596815 -0.7332906 -0.3006048 -0.2183654 -0.2268058 -0.1872942 -0.1521422 -0.0758591 
-0.5282035 -0.5215182 -0.5444490 -0.3897709 -0.2682144 -0.1785277 -0.1328561 -0.1824498 -0.1453178 
-0.5680595 -0.5822325 -0.4597999 -0.4011453 -0.2940212 -0.1721114 -0.1513000 -0.1491470 -0.1225506
: Controller: rectangular_boardsize 7:10
: GPU -1 finishing, processed 3 rows 3 batches
: Cleaned up old neural net and bot
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 7 * 10 exactly
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 7 nnYLen 10
: Initializing board with boardXSize 7 boardYSize 10
: = 
: Controller: kata-set-rules new-zealand
: Changed rules to NewZealand
: = 
: Controller: play b c3
: = 
: Controller: play w c7
: = 
: Controller: play b f7
: = 
: Controller: kata-raw-nn all
: = symmetry 0
whiteWin 0.690885
whiteLoss 0.309115
noResult 0.000000
whiteLead 1.975
whiteScoreSelfplay 2.600
whiteScoreSelfplaySq 150.641
varTimeLeft -1.000
shorttermWinlossError -1.000
shorttermScoreError -1.000
policy
0.000010 0.000014 0.000017 0.000014 0.000014 0.000016 0.000017 
0.000013 0.000021 0.000047 0.000188 0.000062 0.000035 0.000023 
0.000012 0.000025 0.000031 0.010963 0.219546 0.000091 0.000013 
0.000020 0.000019     NAN 0.000240 0.009350     NAN 0.000026 
0.000022 0.000043 0.000056 0.000688 0.075253 0.000122 0.000018 
0.000020 0.000148 0.005326 0.075848 0.007378 0.000134 0.000024 
0.000018 0.000105 0.004531 0.553230 0.023541 0.000068 0.000021 
0.000014 0.000111     NAN 0.004009 0.007827 0.000049 0.000020 
0.000024 0.000032 0.000092 0.000115 0.000128 0.000030 0.000021 
0.000016 0.000020 0.000011 0.000015 0.000013 0.000019 0.000016 
policyPass 0.000001 
whiteOwnership
0.3576331 0.2363918 0.2171397 0.0980499 -0.0173166 -0.1988588 -0.3075440 
0.3717752 0.3257021 0.2655876 0.2194895 -0.0302988 -0.1939640 -0.4003141 
0.4531280 0.3753524 0.4043039 0.1335954 0.2366561 -0.3895475 -0.3951040 
0.3248908 0.4872759 0.7440652 0.0955321 -0.1073775 -0.4937840 -0.4095319 
0.2690805 0.2843151 0.2389281 -0.0204894 0.0464522 -0.4096730 -0.4091078 
0.0320228 -0.1211258 0.0799598 0.1514562 -0.0139161 -0.2392641 -0.2609831 
-0.2365820 -0.2933373 -0.2722558 0.2900148 -0.0812813 -0.1837778 -0.2425437 
-0.3817338 -0.4508418 -0.6553847 -0.4621354 -0.1479077 -0.1431274 -0.1669573 
-0.5616307 -0.5513751 -0.5867277 -0.4323129 -0.2868251 -0.1595443 -0.1744051 
-0.4962800 -0.6002343 -0.5077067 -0.4670361 -0.3534503 -0.2871441 -0.2106264 

symmetry 1
whiteWin 0.718946
whiteLoss 0.281054
noResult 0.000000
whiteLead 2.582
whiteScoreSelfplay 3.477
whiteScoreSelfplaySq 165.527
varTimeLeft -1.000
shorttermWinlossError -1.000
shorttermScoreError -1.000
policy
0.000012 0.000013 0.000014 0.000014 0.000012 0.000017 0.000010 
0.000011 0.000015 0.000043 0.000278 0.000072 0.000032 0.000017 
0.000012 0.000026 0.000035 0.010061 0.308187 0.000099 0.000011 
0.000016 0.000020     NAN 0.000265 0.020652     NAN 0.000025 
0.000015 0.000036 0.000060 0.001296 0.088690 0.000104 0.000014 
0.000013 0.000134 0.020171 0.099353 0.004542 0.000155 0.000026 
0.000017 0.000118 0.004710 0.416009 0.015884 0.000095 0.000023 
0.000014 0.000058     NAN 0.002770 0.005249 0.000047 0.000018 
0.000021 0.000021 0.000043 0.000091 0.000090 0.000025 0.000019 
0.000013 0.000018 0.000011 0.000014 0.000014 0.000018 0.000014 
policyPass 0.000001 
whiteOwnership
0.3659723 0.3260349 0.1826536 0.0581334 0.0325708 -0.1395810 -0.3095496 
0.4362562 0.3408927 0.2996293 0.2001658 -0.0452260 -0.2529528 -0.3796059 
0.4718868 0.3681900 0.4049207 0.2093102 0.3144699 -0.4107530 -0.4722157 
0.3543495 0.4605016 0.7505129 0.0574649 -0.0742328 -0.5240911 -0.4284795 
0.2304715 0.2861344 0.2125119 0.0014990 0.1306599 -0.4222229 -0.3871607 
0.0993869 0.0287297 0.1436577 0.2612703 0.0076887 -0.1897331 -0.2854112 
-0.2197749 -0.1810826 -0.2611547 0.2871174 -0.1040154 -0.1233270 -0.1700282 
-0.3631069 -0.3579507 -0.5071421 -0.3433388 -0.1212201 -0.0803907 -0.0689647 
-0.5121159 -0.4719777 -0.4068956 -0.3705221 -0.1723294 -0.1663207 -0.1432208 
-0.5166386 -0.5575498 -0.4130727 -0.3586604 -0.2646951 -0.3199901 -0.2035572 

symmetry 2
whiteWin 0.680947
whiteLoss 0.319052
noResult 0.000000
whiteLead 1.600
whiteScoreSelfplay 2.697
whiteScoreSelfplaySq 160.377
varTimeLeft -1.000
shorttermWinlossError -1.000
shorttermScoreError -1.000
policy
0.000014 0.000018 0.000020 0.000019 0.000016 0.000015 0.000013 
0.000015 0.000025 0.000059 0.000154 0.000037 0.000029 0.000022 
0.000016 0.000034 0.000056 0.021000 0.191262 0.000063 0.000011 
0.000022 0.000024     NAN 0.000521 0.050724     NAN 0.000022 
0.000020 0.000054 0.000050 0.001995 0.033443 0.000121 0.000015 
0.000016 0.000259 0.022820 0.138250 0.011812 0.000180 0.000021 
0.000017 0.000169 0.016058 0.451717 0.043331 0.000071 0.000016 
0.000017 0.000082     NAN 0.005901 0.008744 0.000045 0.000014 
0.000021 0.000033 0.000065 0.000111 0.000154 0.000027 0.000018 
0.000016 0.000023 0.000012 0.000017 0.000016 0.000019 0.000017 
policyPass 0.000001 
whiteOwnership
0.3164535 0.3102254 0.2461526 0.1438376 0.0301003 -0.0138526 -0.2054172 
0.3603488 0.3259934 0.3210198 0.2982714 -0.0436583 -0.1910697 -0.4216231 
0.4272749 0.3920218 0.4409394 0.1829252 0.1177760 -0.2768134 -0.3814116 
0.3542016 0.4564030 0.7980418 0.1710101 -0.1273901 -0.5028375 -0.4911535 
0.2053279 0.2846949 0.2824370 -0.0503705 0.0315545 -0.4293920 -0.4510271 
0.0482084 -0.0560184 0.0974967 0.2403056 0.0428599 -0.2201716 -0.2539433 
-0.2028499 -0.1915751 -0.2231583 0.3134897 -0.0479169 -0.1473592 -0.1976483 
-0.3602225 -0.4396485 -0.6463931 -0.4445419 -0.0745550 -0.0694299 -0.1059311 
-0.5676939 -0.5379557 -0.5279725 -0.3324947 -0.2717624 -0.0943375 -0.1897886 
-0.5109841 -0.5958549 -0.4748026 -0.3936514 -0.2856832 -0.2403169 -0.1733109 

symmetry 3
whiteWin 0.693356
whiteLoss 0.306644
noResult 0.000000
whiteLead 2.390
whiteScoreSelfplay 3.112
whiteScoreSelfplaySq 163.813
varTimeLeft -1.000
shorttermWinlossError -1.000
shorttermScoreError -1.000
policy
0.000012 0.000019 0.000021 0.000023 0.000017 0.000021 0.000017 
0.000015 0.000023 0.000054 0.000204 0.000060 0.000033 0.000025 
0.000014 0.000031 0.000030 0.007626 0.226143 0.000090 0.000016 
0.000023 0.000029     NAN 0.000372 0.070038     NAN 0.000030 
0.000024 0.000062 0.000070 0.001745 0.051984 0.000108 0.000017 
0.000016 0.000160 0.022508 0.108574 0.002711 0.000099 0.000024 
0.000019 0.000167 0.008255 0.480546 0.008634 0.000051 0.000020 
0.000018 0.000078     NAN 0.004327 0.004119 0.000038 0.000018 
0.000028 0.000033 0.000063 0.000120 0.000176 0.000035 0.000024 
0.000019 0.000028 0.000017 0.000020 0.000021 0.000021 0.000016 
policyPass 0.000001 
whiteOwnership
0.3905719 0.3759236 0.1919469 0.1117735 -0.0087102 -0.2327676 -0.2890364 
0.4260013 0.3431301 0.2491297 0.3183262 -0.0659023 -0.3735400 -0.4653836 
0.4430607 0.4288056 0.4515887 0.1741596 0.1872758 -0.4379148 -0.5182531 
0.3286835 0.4413411 0.8153940 0.1707592 -0.1607017 -0.5631924 -0.5066829 
0.2205451 0.2653887 0.2535683 -0.0330793 -0.0163838 -0.4083107 -0.4273274 
0.0574216 0.0064067 0.1338141 0.2741753 -0.0113257 -0.2427978 -0.2820945 
-0.1915550 -0.2217330 -0.2173441 0.2697799 -0.0691304 -0.1845429 -0.2000688 
-0.3536360 -0.3847444 -0.4341592 -0.4329920 -0.0599904 -0.0794295 -0.0895869 
-0.5168359 -0.5686054 -0.4877899 -0.3629214 -0.1691224 -0.1806623 -0.1739288 
-0.4411286 -0.5640187 -0.4729939 -0.4359884 -0.2829207 -0.2611795 -0.1243465 

symmetry 4
whiteWin 0.690885
whiteLoss 0.309115
noResult 0.000000
whiteLead 1.975
whiteScoreSelfplay 2.600
whiteScoreSelfplaySq 150.641
varTimeLeft -1.000
shorttermWinlossError -1.000
shorttermScoreError -1.000
policy
0.000010 0.000014 0.000017 0.000014 0.000014 0.000016 0.000017 
0.000013 0.000021 0.000047 0.000188 0.000062 0.000035 0.000023 
0.000012 0.000025 0.000031 0.010963 0.219546 0.000091 0.000013 
0.000020 0.000019     NAN 0.000240 0.009350     NAN 0.000026 
0.000022 0.000043 0.000056 0.000688 0.075253 0.000122 0.000018 
0.000020 0.000148 0.005326 0.075848 0.007378 0.000134 0.000024 
0.000018 0.000105 0.004531 0.553230 0.023541 0.000068 0.000021 
0.000014 0.000111     NAN 0.004009 0.007827 0.000049 0.000020 
0.000024 0.000032 0.000092 0.000115 0.000128 0.000030 0.000021 
0.000016 0.000020 0.000011 0.000015 0.000013 0.000019 0.000016 
policyPass 0.000001 
whiteOwnership
0.3576331 0.2363918 0.2171397 0.0980499 -0.0173166 -0.1988588 -0.3075440 
0.3717752 0.3257021 0.2655876 0.2194895 -0.0302988 -0.1939640 -0.4003141 
0.4531280 0.3753524 0.4043039 0.1335954 0.2366561 -0.3895475 -0.3951040 
0.3248908 0.4872759 0.7440652 0.0955321 -0.1073775 -0.4937840 -0.4095319 
0.2690805 0.2843151 0.2389281 -0.0204894 0.0464522 -0.4096730 -0.4091078 
0.0320228 -0.1211258 0.0799598 0.1514562 -0.0139161 -0.2392641 -0.2609831 
-0.2365820 -0.2933373 -0.2722558 0.2900148 -0.0812813 -0.1837778 -0.2425437 
-0.3817338 -0.4508418 -0.6553847 -0.4621354 -0.1479077 -0.1431274 -0.1669573 
-0.5616307 -0.5513751 -0.5867277 -0.4323129 -0.2868251 -0.1595443 -0.1744051 
-0.4962800 -0.6002343 -0.5077067 -0.4670361 -0.3534503 -0.2871441 -0.2106264 

symmetry 5
whiteWin 0.718946
whiteLoss 0.281054
noResult 0.000000
whiteLead 2.582
whiteScoreSelfplay 3.477
whiteScoreSelfplaySq 165.527
varTimeLeft -1.000
shorttermWinlossError -1.000
shorttermScoreError -1.000
policy
0.000012 0.000013 0.000014 0.000014 0.000012 0.000017 0.000010 
0.000011 0.000015 0.000043 0.000278 0.000072 0.000032 0.000017 
0.000012 0.000026 0.000035 0.010061 0.308187 0.000099 0.000011 
0.000016 0.000020     NAN 0.000265 0.020652     NAN 0.000025 
0.000015 0.000036 0.000060 0.001296 0.088690 0.000104 0.000014 
0.000013 0.000134 0.020171 0.099353 0.004542 0.000155 0.000026 
0.000017 0.000118 0.004710 0.416009 0.015884 0.000095 0.000023 
0.000014 0.000058     NAN 0.002770 0.005249 0.000047 0.000018 
0.000021 0.000021 0.000043 0.000091 0.000090 0.000025 0.000019 
0.000013 0.000018 0.000011 0.000014 0.000014 0.000018 0.000014 
policyPass 0.000001 
whiteOwnership
0.3659723 0.3260349 0.1826536 0.0581334 0.0325708 -0.1395810 -0.3095496 
0.4362562 0.3408927 0.2996293 0.2001658 -0.0452260 -0.2529528 -0.3796059 
0.4718868 0.3681900 0.4049207 0.2093102 0.3144699 -0.4107530 -0.4722157 
0.3543495 0.4605016 0.7505129 0.0574649 -0.0742328 -0.5240911 -0.4284795 
0.2304715 0.2861344 0.2125119 0.0014990 0.1306599 -0.4222229 -0.3871607 
0.0993869 0.0287297 0.1436577 0.2612703 0.0076887 -0.1897331 -0.2854112 
-0.2197749 -0.1810826 -0.2611547 0.2871174 -0.1040154 -0.1233270 -0.1700282 
-0.3631069 -0.3579507 -0.5071421 -0.3433388 -0.1212201 -0.0803907 -0.0689647 
-0.5121159 -0.4719777 -0.4068956 -0.3705221 -0.1723294 -0.1663207 -0.1432208 
-0.5166386 -0.5575498 -0.4130727 -0.3586604 -0.2646951 -0.3199901 -0.2035572 

symmetry 6
whiteWin 0.680947
whiteLoss 0.319052
noResult 0.000000
whiteLead 1.600
whiteScoreSelfplay 2.697
whiteScoreSelfplaySq 160.377
varTimeLeft -1.000
shorttermWinlossError -1.000
shorttermScoreError -1.000
policy
0.000014 0.000018 0.000020 0.000019 0.000016 0.000015 0.000013 
0.000015 0.000025 0.000059 0.000154 0.000037 0.000029 0.000022 
0.000016 0.000034 0.000056 0.021000 0.191262 0.000063 0.000011 
0.000022 0.000024     NAN 0.000521 0.050724     NAN 0.000022 
0.000020 0.000054 0.000050 0.001995 0.033443 0.000121 0.000015 
0.000016 0.000259 0.022820 0.138250 0.011812 0.000180 0.000021 
0.000017 0.000169 0.016058 0.451717 0.043331 0.000071 0.000016 
0.000017 0.000082     NAN 0.005901 0.008744 0.000045 0.000014 
0.000021 0.000033 0.000065 0.000111 0.000154 0.000027 0.000018 
0.000016 0.000023 0.000012 0.000017 0.000016 0.000019 0.000017 
policyPass 0.000001 
whiteOwnership
0.3164535 0.3102254 0.2461526 0.1438376 0.0301003 -0.0138526 -0.2054172 
0.3603488 0.3259934 0.3210198 0.2982714 -0.0436583 -0.1910697 -0.4216231 
0.4272749 0.3920218 0.4409394 0.1829252 0.1177760 -0.2768134 -0.3814116 
0.3542016 0.4564030 0.7980418 0.1710101 -0.1273901 -0.5028375 -0.4911535 
0.2053279 0.2846949 0.2824370 -0.0503705 0.0315545 -0.4293920 -0.4510271 
0.0482084 -0.0560184 0.0974967 0.2403056 0.0428599 -0.2201716 -0.2539433 
-0.2028499 -0.1915751 -0.2231583 0.3134897 -0.0479169 -0.1473592 -0.1976483 
-0.3602225 -0.4396485 -0.6463931 -0.4445419 -0.0745550 -0.0694299 -0.1059311 
-0.5676939 -0.5379557 -0.5279725 -0.3324947 -0.2717624 -0.0943375 -0.1897886 
-0.5109841 -0.5958549 -0.4748026 -0.3936514 -0.2856832 -0.2403169 -0.1733109 

symmetry 7
whiteWin 0.693356
whiteLoss 0.306644
noResult 0.000000
whiteLead 2.390
whiteScoreSelfplay 3.112
whiteScoreSelfplaySq 163.813
varTimeLeft -1.000
shorttermWinlossError -1.000
shorttermScoreError -1.000
policy
0.000012 0.000019 0.000021 0.000023 0.000017 0.000021 0.000017 
0.000015 0.000023 0.000054 0.000204 0.000060 0.000033 0.000025 
0.000014 0.000031 0.000030 0.007626 0.226143 0.000090 0.000016 
0.000023 0.000029     NAN 0.000372 0.070038     NAN 0.000030 
0.000024 0.000062 0.000070 0.001745 0.051984 0.000108 0.000017 
0.000016 0.000160 0.022508 0.108574 0.002711 0.000099 0.000024 
0.000019 0.000167 0.008255 0.480546 0.008634 0.000051 0.000020 
0.000018 0.000078     NAN 0.004327 0.004119 0.000038 0.000018 
0.000028 0.000033 0.000063 0.000120 0.000176 0.000035 0.000024 
0.000019 0.000028 0.000017 0.000020 0.000021 0.000021 0.000016 
policyPass 0.000001 
whiteOwnership
0.3905719 0.3759236 0.1919469 0.1117735 -0.0087102 -0.2327676 -0.2890364 
0.4260013 0.3431301 0.2491297 0.3183262 -0.0659023 -0.3735400 -0.4653836 
0.4430607 0.4288056 0.4515887 0.1741596 0.1872758 -0.4379148 -0.5182531 
0.3286835 0.4413411 0.8153940 0.1707592 -0.1607017 -0.5631924 -0.5066829 
0.2205451 0.2653887 0.2535683 -0.0330793 -0.0163838 -0.4083107 -0.4273274 
0.0574216 0.0064067 0.1338141 0.2741753 -0.0113257 -0.2427978 -0.2820945 
-0.1915550 -0.2217330 -0.2173441 0.2697799 -0.0691304 -0.1845429 -0.2000688 
-0.3536360 -0.3847444 -0.4341592 -0.4329920 -0.0599904 -0.0794295 -0.0895869 
-0.5168359 -0.5686054 -0.4877899 -0.3629214 -0.1691224 -0.1806623 -0.1739288 
-0.4411286 -0.5640187 -0.4729939 -0.4359884 -0.2829207 -0.2611795 -0.1243465
: GPU -1 finishing, processed 8 rows 8 batches
: All cleaned up, quitting
