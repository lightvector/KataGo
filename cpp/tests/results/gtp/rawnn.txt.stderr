KataGo v1.16.4
Using TrompTaylor rules initially, unless GTP/GUI overrides this
Initializing board with boardXSize 19 boardYSize 19
Loaded config configs/gtp_example.cfg and/or command-line and query overrides
Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
Model name: g170-b6c96-s175395328-d26788732
GTP ready, beginning main protocol loop
Initializing board with boardXSize 9 boardYSize 9
Changed rules to Chinese
Initializing board with boardXSize 7 boardYSize 10
Changed rules to NewZealand
